### ISSUE TITLE
Phi-phi Pool Earnings Tracker

### ISSUE THREAD

**Issue by Frunzl** [nemosminer/NemosMiner#862]
Hi,

Phi-phi Pool Earnings Tracker doesnt work. Cant figure out why...

Please help!

**Comment by gilby77** [comment]
Remove this pool-pool is without autoexchange now.

**Comment by Frunzl** [comment]
that's not true, it may be a shitty pool, but it does still have auto-exchange

**Comment by Yogurt79** [comment]
![screenshot_1](https://user-images.githubusercontent.com/43872755/49355352-c2c85a80-f6f9-11e8-9fa0-346e0eb84ed1.jpg)


Pool just burns))))
Please correct

**Comment by MrPlusGH** [comment]
https://github.com/MrPlusGH/NPlusMiner/releases/tag/4.5.3

**Comment by MrPlusGH** [comment]
#878 to #888 Pull requests

**Comment by nemosminer** [comment]
thank Mrplus

**Comment by nemosminer** [comment]
"Hi,

Phi-phi Pool Earnings Tracker doesnt work. Cant figure out why...

Please help!"

**Comment by nemosminer** [comment]
auto exchange is working .. first pool i've found to have x21s (RITO) .. price's are back to normal

Earning Tracker is still not showing in console but is be tracked by the monitor server if you use it..

**Comment by nemosminer** [comment]
https://images-ext-2.discordapp.net/external/DIXpzbxcb6U72v-RvGhbGBDfX-AbDprurSbFRaGVy5w/https/i.imgur.com/mgg5yLsh.jpg

**Comment by nemosminer** [comment]
https://images-ext-1.discordapp.net/external/ixtiWrAcCD-Fk6qcQS3h1rne02MG9UJx1GKPoZZfRlw/https/i.imgur.com/cfOMGnsh.jpg?width=859&height=656

**Comment by MrPlusGH** [comment]
Need to update the PoolAPIRef.json for earning Tracker to work.
Update URIs and use "total" instead of "total_earned"
